.github/workflows/integration-tests.yml

```diff
@@ -263,12 +265,12 @@ jobs:
         uses: actions/checkout@ac593985615ec2ede58e132d2e21d2b1cbd6127c # v3.3.0
         with:
           repository: smartcontractkit/chainlink-solana
-          ref: ${{ env.solana_sha }}
+          ref: ${{ env.SOLANA_REF }}
       - name: Build contracts
-        if: needs.changes.outputs.src == 'true'
+        if: ${{needs.changes.outputs.src == 'true' && env.SOLANA_REF != 'develop' }}
```

Comment:
So this never runs?

Reply:
It should run only if
1. there are changes that require full test runs
and
2. we are testing against something other than solana's develop branch.
This can come into play when solana isn't compatible with what is on chainlink develop and we need to switch the solana ref. Mostly seen for release versions of chainlink.